plugins/CoreAdminHome/Commands/InvalidateReportData.php

```diff
@@ -54,6 +62,7 @@ protected function configure()
         $this->addOption('dry-run', null, InputOption::VALUE_NONE, 'For tests. Runs the command w/o actually '
             . 'invalidating anything.');
         $this->addOption('plugin', null, InputOption::VALUE_REQUIRED, 'To invalidate data for a specific plugin only.');
+        $this->addOption('yes', null, InputOption::VALUE_NONE, 'Assume yes if an unrecognized segment is given.');
```

Comment:
Could we maybe clarify this help text as I don't quite get it? could the parameter name be maybe more clear?

Reply:
`--ignore-unrecognized-segment`? or `--quiet`/`--force`?

Reply:
It'd probably be better to just remove the check.